app/assets/javascripts/discourse-markdown-it/src/engine.js

```diff
@@ -0,0 +1,352 @@
+import markdownit from "markdown-it";
```

Comment:
This was previously part of `pretty-text/addon/engines/discourse-markdown-it.js`